hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestMetaRegionLocationCache.java

```diff
@@ -60,7 +60,7 @@ public static void setUp() throws Exception {
     TEST_UTIL.getConfiguration().setInt(HConstants.META_REPLICAS_NUM, 3);
     TEST_UTIL.startMiniCluster(3);
     REGISTRY = ConnectionRegistryFactory.getRegistry(TEST_UTIL.getConfiguration());
-    RegionReplicaTestHelper.waitUntilAllMetaReplicasHavingRegionLocation(
+    RegionReplicaTestHelper.waitUntilAllMetaReplicasAreReady(TEST_UTIL,
```

Comment:
We pass in a TEST_UTIL AND a Configuration? Do we always get the configuration from TEST_UTIL? If so, just pass TEST_UTIL? Otherwise, ignore this comment.